x-pack/plugin/data-streams/src/main/java/org/elasticsearch/xpack/datastreams/mapper/DataStreamTimestampFieldMapper.java

```diff
@@ -201,4 +201,8 @@ public void postParse(ParseContext context) throws IOException {
     protected String contentType() {
         return NAME;
     }
+
+    boolean isEnabled() {
```

Comment:
Could you leave a comment on the pr about why this worked before?

Reply:
We didn't need it before because we were using `MapperTestUtils#assertConflicts` to test updates, and that checks that a conflict doesn't occur but doesn't check that it has had any effect.  MapperTestCase's parameter checks do both, so we need a way of finding out what the `enabled` value is, given a DocumentMapper.